rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -686,7 +688,10 @@ rmw_ret_t rmw_init()
   return RMW_RET_OK;
 }
 
-rmw_node_t * rmw_create_node(const char * name, const char * namespace_, size_t domain_id)
+rmw_node_t * create_node(
+  const char * name,
+  const char * namespace_,
+  ParticipantAttributes participantParam)
```

Comment:
good catch, just copy-n-paste the original variable without renaming it. Will fix